### PR TITLE
Removed redundant "Documentation" anchor

### PR DESCRIPTION
**Pull request by sinkswim**
Removed redundant "Documentation" anchor that hreffed to '/getting-started' already pointed by Getting Started.

### CODE REVIEW

**Comment by JKAussieSkater** [comment]
@sinkswim, you make a really good point with this pull request. In general, redundancy is bad, and duplicates are unnecessary. I didn't realise those two links were different until you brought it to my attention!

The reason I don't want to merge this pull request is because of the distinct usefulness of them both:

When I was a Grunt newbie . . . | When I was a Grunt beginner . . .
------------ | -------------
I wanted a _beginners guide_. | I needed to _refer to the manual_.
I navigated to the Grunt website | I navigated to the Grunt website 
I saw the **"Getting Started"** link in the navbar | I saw the **"Documentation"** link in the navbar
I clicked it right away | I clicked it right away

In other words, the navigation bar helped me to find what I needed straight away, without hunting for the links my brain was looking for. I'm certain others have also used the navigation bar in the way I had.

----

Perhaps an alternative solution is to create a new "Documentation" page, which has an index where certain features can be found in the docs (rather than referring to "Getting Started" and navigating the side-bar links).

What do you think?

**Comment by sinkswim** [comment]
@JKAussieSkater I've used the navigation bar the same way you have described, so I get what you mean.
Still, I have no sympathy for redundancy ;) so your solution sounds good to me!

**Comment by JKAussieSkater** [comment]
Thanks!

Okay, lets create a [Documentation Overview](https://github.com/gruntjs/grunt-docs/issues/156) page instead!
